src/target/stm32l4.c

```diff
@@ -743,29 +735,24 @@ static bool stm32l4_cmd_option(target *t, int argc, char *argv[])
 		tc_printf(t, "STM32WLxx options not implemented!\n");
 		return false;
 	}
-	static const uint32_t g4_values[11] = {
-		/* SEC_SIZE1 occupies 9 bits on G49/G4A (cat 4),
+	static const uint32_t g4_values[11] = {/* SEC_SIZE1 occupies 9 bits on G49/G4A (cat 4),
```

Comment:
this comment messed the formatting slightly

Reply:
This is addressed in 1b709240 on fix/target-cleanup